qa/automation/ScoringPortal.Tests.Scenarios/UsoltsevaL/Sign_in.feature

```diff
@@ -0,0 +1,184 @@
+    And user fills "accountfortesting@mail.ru" in "Email or phone" text field
+    And user clicks "Next" button
+    And user fills in "Khjg4865" in "Enter your password" text field
+    And user clicks "Next" button
+    Then Error message "!Wrong password. Try again or click Forgot password to reset it" is displayed and text field "Enter your password" has a red frame
+@tc:9
+Scenario: 1.1.1.1_09 Logging to the application with not the Google Account (reduced one)
+    Given user didn't sign in Google Account in page https://accounts.google.com
+    And user has a mail account accountfortesting@mail.ru with assigned phone number +79658905895, password is Khjg4865
+    When user clicks on "Sign in with Google " button
+    And user fills "accountfortesting" in "Email or phone" text field
+	Then Error message "Couldn't find your Google Account" is displayed and text field "Email or phone" has a red frame
+@tc:10
+Scenario: 1.1.1.1_10 Logging to the application with incorrect format of phone number (without +)
+    Given user didn't sign in Google Account in page https://accounts.google.com
+    When user clicks on "Sign in with Google " button
+    And user fills "79658905735" in "Email or phone" text field
+    And user clicks "Next" button
+    Then Error message "!Couldn’t find your Google Account. Try using your email address instead" is displayed and text field "Email or phone" has a red frame
+@tc:11
+Scenario: 1.1.1.1_11 Logging to the application with incorrect format of phone number (without +7)
+    Given user didn't sign in Google Account in page https://accounts.google.com
+    When user clicks on "Sign in with Google " button
+    And user fills "9658905735" in "Email or phone" text field
+    And user clicks "Next" button
+    Then Error message "!Couldn’t find your Google Account. Try using your email address instead" is displayed and text field "Email or phone" has a red frame
+@tc:12
+Scenario: 1.1.1.1_12 Logging to the application with empty "Email or phone" field
+    Given user didn't sign in Google Account in page https://accounts.google.com
+    When user clicks on "Sign in with Google " button
+    And user leaves empty "Email or phone" text field
+    And user clicks "Next" button
+    Then Error message "!Enter an email or phone number" is displayed and text field "Email or phone" has a red frame
+@tc:13
+Scenario: 1.1.1.1_13 Logging to the application with wrong PW
+    Given user didn't sign in Google Account in page https://accounts.google.com
+    When user clicks on "Sign in with Google " button
+    And user fills "accountfortesting@gmail.com" in "Email or phone" text field
+    And user clicks "Next" button
+    And user fills in "Po0987$!" in "Enter your password" text field
+    And user clicks "Next" button
+    Then Error message "!Wrong password. Try again or click 'Forgot password' to reset it" is displayed and text field "Enter your password" has a red frame
+@tc:14
```

Comment:
Thank you! It was a great job! 
For more complete coverage it will be better to add some tests (e.g. when user forgot his password, email or doesn't have a Goggle account at all - he should be redirected to other pages)